resources/wikia/modules/iframeWriter.js

```diff
@@ -35,6 +36,10 @@ define('wikia.iframeWriter', [
 		}
 
 		iframe.onload = function () {
+			if (loaded) {
```

Comment:
What do you think about leaving a nice comment in which we blame IE and describe the issue?